docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -7,21 +7,82 @@ Fairness in Machine Learning
+mutually exclusive and often exacerbate one another. Therefore, we define
+whether an AI system is behaving unfairly in terms of its impact on people —
+i.e., in terms of harms — and not in terms of specific causes, such as
+societal biases. Because we define unfairness in this way, we avoid using the
+word *bias* to refer to the unfair behavior of an AI system. Also *bias* is
+ambiguous and means different things to different communities – e.g.,
+statistical bias vs. societal biases. For this reason, we talk about
+*fairness issues* or *fairness-related harms* rather than *bias*, unless
+we are very specifically referring to societal biases (or statistical bias or
+some other definition of bias). Given the many complex sources of unfairness,
+it is not possible to fully *debias* a system or to guarantee fairness; the
+goal is to detect and to mitigate fairness-related harms as much as possible.
+For this reason, don’t use words like *debias*, *unbiased*, *solve* – they set
+up unrealistic expectations; use words like *mitigate*, *address*,
+*prioritize*, *detect*, *identify*, *assess* instead.
+Besides that, it's best to be specific about the :ref:`types_of_harms`
+while acknowledging that the types are not mutually exclusive.
+Fairness-related harms can have varying severities, but the cumulative impact
+of even *non-severe* harms can be extremely burdensome or make people feel
+singled out or undervalued. They can affect both the people who will use an AI
+system and the people who will be directly or indirectly affected by the
+system, either by choice or not. For this reason, when talking about the
+people who might be harmed by a system, we talk about *stakeholders* instead
+of *users*, and don’t just focus on demographic groups (e.g., groups defined
+in terms of race, gender, age, disability status, skin tone, and their
+intersections) or groups that are protected by anti-discrimination laws – the
+most relevant groups may be context specific.
+Stakeholders can belong to overlapping or intersectional groups – e.g.,
+different combinations of race, gender, and age – and considering each group
+separately may obscure harms.
+
+.. note::
+
+  Fairness-related harms can be (re-)introduced at every stage of the AI
+  development and deployment lifecycle and can arise due to any component of
+  an AI system, not just datasets or models – e.g., task definitions, user
+  experiences, evaluation metrics, or deployment contexts.
+
+  AI systems (and technology in general) are never *neutral* –
+  all AI systems necessarily reflect the assumptions, priorities, and values
+  of the people involved in their development and deployment.
+
+Prioritizing fairness in AI systems is a sociotechnical challenge – it is not
+something that can be accomplished via purely technical methods (or purely
+social methods, for that matter). There is no software tool that will *solve*
+fairness in all AI systems – this is not to say that software tools don’t have
```

Comment:
I'd suggest ending the sentence at the hyphen.

Reply:
Miro will rewrite this section of the user guide in a PR of his own, so I'll just put the list into the contributor guide. Reverting the last few commits and adjusting the lists according to earlier feedback.